containers/provision/centos:8.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+dnf install -y 'dnf-command(config-manager)' \
+	       'dnf-command(builddep)' \
+	       make \
+	       rpm-build \
+	       createrepo_c \
+	       valgrind \
+	       git-daemon \
+	       python3-flask \
+	       python3-requests \
+	       python3-pytest \
+	       python3-six \
+	       procps-ng
```

Comment:
Why do we need `procps-ng`? I guess it should be running without it

Reply:
Left over from the pidof/pgrep saga. I'll remove it from Fedora too.